specs/phase0/p2p-interface.md

```diff
@@ -752,8 +755,10 @@ Clients MUST keep a record of signed blocks seen on the epoch range
 where `current_epoch` is defined by the current wall-clock time,
 and clients MUST support serving requests of blocks on this range.
 
-Peers that are unable to reply to block requests within the
-`MIN_EPOCHS_FOR_BLOCK_REQUESTS` epoch range MAY get descored or disconnected at any time.
+Peers that are unable to reply to block requests within the `MIN_EPOCHS_FOR_BLOCK_REQUESTS`
+epoch range SHOULD respond with error code `3: **ResourceUnavailable**`.
```

Comment:
The formatting is weird here, bold word in code block

Reply:
ah, right